test/time_helper_test.rb

```diff
@@ -0,0 +1,140 @@
+require 'test_helper'
+
+describe Hanami::Helpers::TimeHelper do
+  before do
+    @view = TimeView.new
+  end
+
+  let(:date)   { Time.now }
+
+  let(:minute) { 60 }
+  let(:hour)   { 60 * minute }
+  let(:day)    { 24 * hour }
+  let(:week)   { 7  * day }
+  let(:month)  { 4  * week }
+  let(:year)   { 12 * month }
+
+  describe '#in_words' do
+    describe 'with from date' do
+      let(:date_from) { Time.now + 1 * hour }
+      it { @view.created_at(date, date_from).must_equal 'a hour ago' }
```

Comment:
should be `an hour` (since "hour" sounds like "our", it's treated as though it begins with a vowel... silly english 🙃)